duckduckgo/Android
# AppTP: Change endpoint for blocklist to pull new format

Task/Issue URL: https://app.asana.com/0/0/1204239971719179/f

### Description
Now that we're using the cumulative blocklist for AppTP, we've decided to use a new endpoint to support the new list format and any future updates. We'll maintain the old endpoint, without updates, for users on earlier versions.

### Steps to test this PR
The endpoint is up & running with the old (current) list, but in the new format (which I'll alter today to include entity scores).

To test:
Install app from `develop`, enable AppTP & let block some trackers
Update app from this branch, verify that database transitions smoothly without trying to migrate to a new version
